meta-arm/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -0,0 +1 @@
+require ${@bb.utils.contains('MACHINE_FEATURES', 'efi', 'systemd-uefi-secureboot.inc', '', d)}
```

Comment:
MACHINE_FEATURE is "efi" but .inc file name still has "uefi-secureboot". Maybe use systemd-efi.inc file name to keep these consistent.

Reply:
Good catch. Fixed. Thanks!